src/main/kotlin/no/nav/tilleggsstonader/sak/behandling/fakta/BehandlingFaktaDto.kt

```diff
@@ -12,14 +12,28 @@ import no.nav.tilleggsstonader.sak.felles.domain.BarnId
 import java.time.LocalDate
 import java.time.LocalDateTime
 
-data class BehandlingFaktaDto(
-    val søknadMottattTidspunkt: LocalDateTime?,
-    val hovedytelse: FaktaHovedytelse,
-    val aktivitet: FaktaAktivtet,
-    val barn: List<FaktaBarn>,
-    val dokumentasjon: FaktaDokumentasjon?,
-    val arena: ArenaFakta?,
-)
+sealed class BehandlingFaktaDto(
+    open val søknadMottattTidspunkt: LocalDateTime?,
+    open val hovedytelse: FaktaHovedytelse,
+    open val dokumentasjon: FaktaDokumentasjon?,
+    open val arena: ArenaFakta?,
+) {
+    data class BehandlingFaktaDtoTilsynBarn(
+        override val søknadMottattTidspunkt: LocalDateTime?,
+        override val hovedytelse: FaktaHovedytelse,
+        override val dokumentasjon: FaktaDokumentasjon?,
+        override val arena: ArenaFakta?,
+        val aktivitet: FaktaAktivtet?,
```

Comment:
Hvorfor endres disse til nullable? 
```suggestion
        val aktivitet: FaktaAktivtet,
```

Reply:
[213c43a](https://github.com/navikt/tilleggsstonader-sak/pull/452/commits/213c43a56033477e259174422c5c259116191f65) Fikset